robots/simple_humanoid_description/urdf/simple_humanoid.urdf

```diff
@@ -14,6 +14,22 @@
       <mass value="27"/>
       <inertia ixx="1" ixy="0" ixz="0" iyy="1" iyz="0" izz="1" />
     </inertial>
+    <collision name="test">
+      <geometry>
+        <cylinder radius="1" length="1"/>
+      </geometry>
+    </collision>
```

Comment:
Is this a mistake?
Btw, I am not sure if we can add two collision objects in the URDF.

Reply:
We can have multiple collision objects, and multiple visual objects.

Reply:
Ok, thanks for the clarification.